requirements-client.txt

```diff
@@ -14,6 +14,7 @@ httpx[http2] >= 0.23, != 0.23.2
 importlib_metadata >= 4.4; python_version < '3.10'
 jsonpatch >= 1.32, < 2.0
 jsonschema >= 4.0.0, < 5.0.0
+opentelemetry-api
```

Comment:
this should be put in https://github.com/PrefectHQ/prefect/blob/main/requirements-otel.txt

Reply:
sorry i think this is OK in client; but it needs to be removed from `requirements` and include upper / lower bounds as in Collin's PR https://github.com/PrefectHQ/prefect/pull/16010/files